apps/address-service/README.md

```diff
@@ -1,3 +1,21 @@
+# Description
+It's a separate service for address processing.
+
+Each property is presented as the Address model instance. Each address may be found by different strings, so several AddressSource model instances may belong to each address.
+
+## Models
+- **Address**. A model containing data on the particular building's address.
+- **AddressSource**. A model containing data on the particular building's address origin.
+- **AddressInjection**. Addresses that do not exist in external providers.
+
+## Endpoints
+
+### /suggest
+Used to get a list of suggestions while the user trying to add a new property. There is an ability to use different suggestions providers. If some property still not exists in external providers, we may inject proper data before the answer is sent to the response. It may be reached by adding new AddressInjection model via admin interface.
+
+### /search
+Used to search addresses. This endpoint utilizes search plugins under the hood. The first not-empty plugin's result will send to the response.
```

Comment:
What the difference between `/suggest`. Where search is performed?

Reply:
`/suggest` returns a list of suggestions got from external providers + internal injections.
`/search` should be used to get info about a single address by source (it's a cache) or some uuid.

Reply:
Could you add this into comments:)

Reply:
ok